hammett/core/application.py

```diff
@@ -58,7 +58,13 @@ def __init__(
         persistence: 'BasePersistence[UD, CD, BD] | None' = None,
         states: 'States | None' = None,
     ) -> None:
-        """Initialize an application object."""
+        """Initialize an application object.
+
+        Raises
+        ------
+            TokenIsNotSpecified: If `TOKEN` attribute of settings is not specified.
```

Comment:
`TokenIsNotSpecified: If the `TOKEN` attribute in the settings is not specified.`